e2e/tests/functional/plugins/displayLayout/displayLayout.e2e.spec.js

```diff
@@ -598,9 +598,37 @@ test.describe('Display Layout', () => {
     });
 
     // Verify filtering is working correctly
-    // Wait for some data to populate
-    // eslint-disable-next-line playwright/no-wait-for-timeout
-    await page.waitForTimeout(2000); // Wait for a few state changes
+
```

Comment:
I think Playwright gone done this already - https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-count

Reply:
well then... glad I did ALL THAT. Haha, let me switch this out and test to make sure.

Reply:
Actually, going back, I remember why I did this. The table is adding and removing rows basically at the same time, so I can't go by count, because the rows aren't being added. So what I'm doing here is observing added nodes to the `table` child list (`tr`'s would count). I wait for 20 rows to be added, then I verify there are non-filtered rows shown and filtered rows NOT shown.

Reply:
I think I understand, you want to count 20 add events, not 20 total rows.